doc/tls-configuration.md

```
@@ -163,6 +348,139 @@ FS_KEMs: OK X25519MLKEM768
 ```
 
 
+---
+
+## Internal Connectivity Analysis
+
+This section analyses the TLS settings used on each internal connection path and evaluates whether any server-side TLS configuration set via the NooBaa CR could break internal connectivity.
+
+### Connection Paths
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)